backend/docs/development_docs/api_requests_per_screen.rst

```diff
@@ -70,54 +76,109 @@ SOLR will return data in JSON format and according to the following template: ::
-        "id":"38bbb6ad-d31f-4381-aa88-61be37f70d8c",
-        "_version_":1616659562091249664}]
+        {
+           "document":"<very long string omitted>",
+        "id":"8284b4b5-118a-4cbf-aa7c-f08ae2ca391c",
+        "_version_":1617934905120915456}    
+    ]
      }}
 
+In this output, the fields ``id`` and ``_version_`` are not used. The
+actual contents of the found object are in the ``document`` field, which
+is a JSON encoded string. 
+
+If the ``document`` field of any of these results is parsed, we may, e.g., get an organisation
+unit that looks like this: ::
+    {
+    "associated": [
+    [
+      "Amin Laurvig Deichgraeber", 
+      "03ecfcfd-33ab-4d9a-bc46-df4fb9e60ceb", 
+      "Medhj\u00e6lper", 
+      "Teknisk Servicemedarb."
+    ]
+    ], 
+    "departments": [], 
+    "employees": [
+    [
+      "Jesper Olesen Jepsen", 
+      "839ee38a-ce78-4ef7-989c-8c0fa587857a", 
+      "Ansat", 
+      "P\u00e6dagogmedhj\u00e6lper"
+    ], 
+    [
+      "Jesper  Overmark Jensen", 
+      "06410197-b6b0-48c2-a07c-2506dd711862", 
+      "Ansat", 
+      "Timel\u00f8nnet l\u00e6rer"
+    ], 
+    [
+      "Esther S\u00f8rensen Haugsted", 
+      "e2fc5e65-2cf0-4e1c-b902-33d125a59267", 
+      "Ansat", 
+      "Personalekonsulent"
+    ], 
+    [
+      "Camilla  Hansen", 
+      "2b76a21f-e803-48e3-b0ad-90911910a4f1", 
+      "Ansat", 
+      "L\u00e6rer/Overl\u00e6rer"
+    ], 
+    [
+      "Linda Bjerring Hansen", 
+      "a60fcdaa-9487-4cae-94f4-9cead9d55d75", 
+      "Ansat", 
+      "Udviklingskonsulent"
+    ], 
+    [
+      "Kadiatou Schultz Petersen", 
+      "fa42289b-745c-46df-9289-a9f4c155ef19", 
+      "Ansat", 
+      "P\u00e6dagoisk vejleder"
+    ], 
+    [
+      "Margrethe Neess Borup", 
+      "5a18ce2b-267a-475d-a781-7f6b714b4e46", 
+      "Ansat", 
+      "Teknisk Servicemedarb."
+    ]
+    ], 
+    "locations": [
+    [
+      "DAR", 
+      "Vesterklit 9, L\u00f8nstrup, 9800 Hj\u00f8rring"
+    ]
+    ], 
+    "managers": [
+    [
+      "Direkt\u00f8r", 
+      "Margrethe Neess Borup", 
+      "5a18ce2b-267a-475d-a781-7f6b714b4e46"
+    ]
+    ], 
+    "name": "Budget og Planl\u00e6gning", 
+    "parent": "40644200-b3f1-42d4-8752-8dab581d5b23", 
+    "uuid": "d4f9af18-aacd-48de-aa78-5f29cb23d716"
+    }
+
 Of course, there will be the number of ``docs`` corresponding to the
 ``numFound`` parameter in the ``response``.
 
 In the present example, there is only one document and the name to be
-dislayed is simply "Hjørring".  The members are, apart from the obvious
-ones, to be interpreted like this:
+dislayed is "Budget og Planlægning".  The members are, apart from the
+obvious ones, to be interpreted like this:
 
 * ``locations`` are addresses and can be of type ``DAR``, ``EMAIL``
-  and ``PHONE``. The list contains a list of addresses that are to be
-  interpreted as couples, *(type, value)*.
+  and ``PHONE``. The list contains a list of addresses; each address is
+  itself a couple,  *[type, value]*.
```

Comment:
I'm fairly certain you mean a [pair](https://www.merriam-webster.com/dictionary/pair), not a [couple](https://www.merriam-webster.com/dictionary/couple). (Unless the type really is married to the value? 😁) Strictly speaking, the addresses aren't two-tuples or pairs — given that neither exists in the JSON realm — but rather two-element lists or arrays.

Furthermore, you should probably expand on what _`type`_ and _`value`_ are.

Reply:
Yes, done!